app/models/chat_channel.rb

```diff
@@ -112,6 +112,10 @@ def pusher_channels
     end
   end
 
+  def chanel_users_ids
```

Comment:
typo here, should be `channel_users_ids`

Reply:
Done